src/images/minio.rs

```diff
@@ -0,0 +1,88 @@
+use crate::{core::WaitFor, Image, ImageArgs};
+use std::collections::HashMap;
+
+const NAME: &str = "minio/minio";
+const TAG: &str = "latest";
```

Comment:
Is this some Amazon campatibility thing again? I'd love to use fixed tags here.

Reply:
I think they can coexist 🤔 
The tag is quite an important part and it's easier to specify it in the constructor than to create a wrapper.
```rs
// RunnableImage approach
let image = RunnableImage::from(images::minio::MinIO::default())
        .with_tag("latest");
        
// Direct approach        
let image = images::minio::MinIO::with_tag("latest");

```

If you disagree, let me know, I'll remove the constructors. No problems.

Reply:
I agree, fixed the tag